app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError, OutdatedVaccineError)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:

```

Comment:
use get() method when checking the presence or accessing the value of a key in all your if statements